site/examples/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+class Examples extends React.Component {
+  render() {
+
+    function renderChildrenAsNav(child, i) {
+      return (
+        <div key={i}>
+          <a
+            className='block link'
+            href={child.route}
+          >{child.name}</a>
+        </div>
+      );
+    }
+
+    return (
+      <div>
+        <div className='my24'>
+          <h1 className='txt-subhead txt-bold my24'>Examples</h1>
+          <p>Assembly isn't useful until you know how to construct the markup with class names to get the design you want. The links below feature a variety of snippets that cover common component and layout interfaces.</p>
```

Comment:
I would rephrase this to be more positive? Starting a sentence with "Assembly isn't useful..." seems weird.